erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -741,4 +741,14 @@ void WebRtcConnection::setTransport(std::shared_ptr<Transport> transport) {  //
   bundle_ = true;
 }
 
+std::pair<RTPExtensionsMap, RTPExtensionsMap> WebRtcConnection::getSourceExtensionMap() {
+  return std::make_pair(extension_processor_.getVideoExtensionMap(), extension_processor_.getAudioExtensionMap());
```

Comment:
as I said before I think getSourceExtensionMap should be also async

Reply:
You're right :+1: but I think it would be better to put just a mutex in getter, because two WebRtcConnections may have two different workers and be executed on different threads.